### PR TITLE
Metadata review - PRESF

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updates to PRESF-B and -C CSVs per metadata review project.

### CODE REVIEW

File: calibration/PRESFB/CGINS-PRESFB-01352__20150228.csv [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+26P71826-1352,CC_pressure_offset_calibration_coefficient,-3.330000e-01,
+26P71826-1352,CC_slope_correction_factor,1,constant
+26P71826-1352,CC_t1,2.787143e+01,
+26P71826-1352,CC_t2,4.900260e-01,
+26P71826-1352,CC_t3,1.718162e+01,
+26P71826-1352,CC_t4,1.343469e+01,
+26P71826-1352,CC_u0,5.825881,
+26P71826-1352,CC_y1,-3.854568e+03,
+26P71826-1352,CC_y2,-1.065269e+04,
+26P71826-1352,CC_y3,0,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01352__20150228.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01352__20160714.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26P71826-1352,CC_b,2796.2,
+26P71826-1352,CC_b,2796.2,source_file QCT 3305-00105-00038-A.zip > 3305-00105-00038-A.cap
 26P71826-1352,CC_c1,2011.033,
 26P71826-1352,CC_c2,-53.76979,
 26P71826-1352,CC_c3,-3419.788,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01352__20160714.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01352__20170724.csv [hunk 3]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26P71826-1352,CC_b,2796.2,
+26P71826-1352,CC_b,2796.2,source_file QCT 3305-00105-00047-A.zip > 3305-00105-00047-A.cap
 26P71826-1352,CC_c1,2011.033,
 26P71826-1352,CC_c2,-53.76979,
 26P71826-1352,CC_c3,-3419.788,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01352__20170724.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01386__20140228.csv [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+26P76848-1386,CC_pressure_offset_calibration_coefficient,-0.217,
+26P76848-1386,CC_slope_correction_factor,1,constant
+26P76848-1386,CC_t1,27.63804,
+26P76848-1386,CC_t2,0.427875,
+26P76848-1386,CC_t3,17.7624,
+26P76848-1386,CC_t4,21.86445,
+26P76848-1386,CC_u0,5.810283,
+26P76848-1386,CC_y1,-3944.121,
+26P76848-1386,CC_y2,-10737.63,
+26P76848-1386,CC_y3,0,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01386__20140228.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01386__20151217.csv [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
-26P76848-1386,CC_offset_correction_factor,0,
+26P76848-1386,CC_offset_correction_factor,0,constant
 26P76848-1386,CC_pressure_offset_calibration_coefficient,-0.219,
 26P76848-1386,CC_slope_correction_factor,1,Constant
 26P76848-1386,CC_t1,27.63804,
-26P76848-1386,CC_t2,0.427875,Constant
+26P76848-1386,CC_t2,0.427875,
 26P76848-1386,CC_t3,17.7624,
 26P76848-1386,CC_t4,21.86445,
 26P76848-1386,CC_u0,5.810283,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01386__20151217.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01386__20170202.csv [hunk 6]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26P76848-1386,CC_b,2796.2,cal coeffs from QCT 3305-00105-00044-A.zip > 3305-00105-00044-23032017.txt
+26P76848-1386,CC_b,2796.2,source_file QCT 3305-00105-00044-A.zip > 3305-00105-00044-23032017.txt
 26P76848-1386,CC_c1,2.066838e+03,
 26P76848-1386,CC_c2,-1.270562e+01,
 26P76848-1386,CC_c3,-3.470497e+03,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01386__20170202.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01387__20140228.csv [hunk 7]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26P76848-1387,CC_b,2796.2,
+26P76848-1387,CC_b,2796.2,source_file QCT 3305-00105-00011-A.zip > 3305-00105-00011-11042014.cap
 26P76848-1387,CC_c1,2094.369,
 26P76848-1387,CC_c2,15.74913,
 26P76848-1387,CC_c3,-3649.47,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01387__20140228.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01387__20151217.csv [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+26P76848-1387,CC_pressure_offset_calibration_coefficient,-0.2777,
+26P76848-1387,CC_slope_correction_factor,1,Constant
+26P76848-1387,CC_t1,27.67723,
+26P76848-1387,CC_t2,0.390712,
+26P76848-1387,CC_t3,18.05891,
+26P76848-1387,CC_t4,26.15364,
+26P76848-1387,CC_u0,5.828209,
+26P76848-1387,CC_y1,-3920.251,
+26P76848-1387,CC_y2,-10474.77,
+26P76848-1387,CC_y3,0,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01387__20151217.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01387__20170724.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26P76848-1387,CC_b,2796.2,
+26P76848-1387,CC_b,2796.2,source_file QCT 3305-00105-00046-A.zip > 3305-00105-00046-A.cap
 26P76848-1387,CC_c1,2094.369,
 26P76848-1387,CC_c2,15.74913,
 26P76848-1387,CC_c3,-3649.47,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01387__20170724.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01392__20141021.csv [hunk 10]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26P-1392,CC_b,2796.2,
+26P-1392,CC_b,2796.2,source_file 3305-00105-00018-A.zip > 3305-00105-00018-A.cap
 26P-1392,CC_c1,2064.659,
 26P-1392,CC_c2,14.16356,
 26P-1392,CC_c3,-3193.857,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01392__20141021.csv]: Looks good.

File: calibration/PRESFB/CGINS-PRESFB-01392__20170104.csv [hunk 11]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26P-1392,CC_b,2796.2,
+26P-1392,CC_b,2796.2,source_file QCT 3305-00105-00045-A.zip > 3305-00105-00045-23032017.txt
 26P-1392,CC_c1,2064.659,
 26P-1392,CC_c2,14.16356,
 26P-1392,CC_c3,-3193.857,
```
Comment (snwhite) [inline calibration/PRESFB/CGINS-PRESFB-01392__20170104.csv]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01388__20140310.csv [hunk 12]
```diff
@@ -1,14 +1,14 @@
...
 26P76848-1388,CC_d2,0,
 26P76848-1388,CC_m,12582.9,
-26P76848-1388,CC_offset_correction_factor,0,
+26P76848-1388,CC_offset_correction_factor,0,constant
 26P76848-1388,CC_pressure_offset_calibration_coefficient,-0.4742,
-26P76848-1388,CC_slope_correction_factor,1,
+26P76848-1388,CC_slope_correction_factor,1,constant
 26P76848-1388,CC_t1,30.36646,
 26P76848-1388,CC_t2,0.74353,
 26P76848-1388,CC_t3,50.42604,
```
Comment (snwhite) [inline calibration/PRESFC/CGINS-PRESFC-01388__20140310.csv]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01388__20150929.csv [hunk 13]
```diff
@@ -1,14 +1,14 @@
...
 26P76848-1388,CC_d2,0,
 26P76848-1388,CC_m,12582.9,
-26P76848-1388,CC_offset_correction_factor,0,
+26P76848-1388,CC_offset_correction_factor,0,constant
 26P76848-1388,CC_pressure_offset_calibration_coefficient,-0.6046,
-26P76848-1388,CC_slope_correction_factor,1,
+26P76848-1388,CC_slope_correction_factor,1,constant
 26P76848-1388,CC_t1,30.36646,
 26P76848-1388,CC_t2,0.74353,
 26P76848-1388,CC_t3,50.42604,
```
Comment (snwhite) [inline calibration/PRESFC/CGINS-PRESFC-01388__20150929.csv]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01400__20141125.csv [hunk 14]
```diff
@@ -1,14 +1,14 @@
...
 26-1400,CC_d2,0,
 26-1400,CC_m,12582.9,
-26-1400,CC_offset_correction_factor,0,
+26-1400,CC_offset_correction_factor,0,constant
 26-1400,CC_pressure_offset_calibration_coefficient,-0.3878,
-26-1400,CC_slope_correction_factor,1,
+26-1400,CC_slope_correction_factor,1,constant
 26-1400,CC_t1,30.35705,
 26-1400,CC_t2,1.204163,
 26-1400,CC_t3,48.01965,
```
Comment (snwhite) [inline calibration/PRESFC/CGINS-PRESFC-01400__20141125.csv]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01400__20151217.csv [hunk 15]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26-1400,CC_b,838.8,
+26-1400,CC_b,838.8,source_file QCT 3305-00105-00032-A.zip > 3305-00105-00032-A.cap
 26-1400,CC_c1,-4538.281,
 26-1400,CC_c2,187.5092,
 26-1400,CC_c3,13630.6,
```
Comment (snwhite) [inline calibration/PRESFC/CGINS-PRESFC-01400__20151217.csv]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01400__20170725.csv [hunk 16]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-26-1400,CC_b,838.8,
+26-1400,CC_b,838.8,source_file QCT 3305-00105-00048-A.zip > 3305-00105-00048-11082017.cap
 26-1400,CC_c1,-4538.281,
 26-1400,CC_c2,187.5092,
 26-1400,CC_c3,13630.6,
```
Comment (snwhite) [inline calibration/PRESFC/CGINS-PRESFC-01400__20170725.csv]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01401__20141125.csv [hunk 17]
```diff
@@ -1,16 +1,16 @@
...
-26-1401,CC_offset_correction_factor,0,
+26-1401,CC_offset_correction_factor,0,constant
 26-1401,CC_pressure_offset_calibration_coefficient,-0.3947,
 26-1401,CC_slope_correction_factor,1,Constant
 26-1401,CC_t1,30.38801,
-26-1401,CC_t2,1.029573,Constant
+26-1401,CC_t2,1.029573,
 26-1401,CC_t3,45.59356,
 26-1401,CC_t4,114.0129,
 26-1401,CC_u0,5.828481,
```
Comment (snwhite) [inline calibration/PRESFC/CGINS-PRESFC-01401__20141125.csv]: Looks good.

File: calibration/PRESFC/CGINS-PRESFC-01400__20170725.csv (hunk 16 above)
Comment (cdobs) [inline calibration/PRESFC/CGINS-PRESFC-01400__20170725.csv]: CD

File: calibration/PRESFC/CGINS-PRESFC-01401__20141125.csv (hunk 17 above)
Comment (cdobs) [inline calibration/PRESFC/CGINS-PRESFC-01401__20141125.csv]: CD

**Pull request merged**